edge-functions/maintenance-page/pages/maintenance.tsx

```diff
@@ -7,8 +7,10 @@ function Home() {
   return (
     <Page className="flex flex-col gap-12">
       <section className="flex flex-col gap-6">
-        <Text variant="h1">Something went wrong 😓</Text>
-        <Text>You can try to refresh the page to see if its fixed.</Text>
+        <Text variant="h1">Under Maintenance</Text>
```

Comment:
I changed the wording hear to make clear that this is indeed the maintenance page, as I got confused the first time I saw the demo and thought something about the demo itself went wrong